config.json

```diff
@@ -13,6 +13,15 @@
 
       ]
     },
+{
+      "slug": "rail-fence-cipher",
+      "core": true,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
```

Comment:
Would you be able to estimate a difficulty and add topics? If not, we can open a separate issue for that once this gets merged.

Reply:
Also `core` should be false. We've already decided a set of core exercises and we probably don't want to add to them at this point. See [this issue](https://github.com/exercism/java/issues/707) for our discussion about which exercises should be core exercises. Also [this issue](https://github.com/exercism/java/issues/758) and [this PR](https://github.com/exercism/java/pull/760) where we changed the `config.json` structure after finding out exercises should only be unlocked by core exercises.

If you want @LeonDsouza, you could look through the core exercises (listed in [this PR](https://github.com/exercism/java/pull/760)) and choose want you think should unlock this one. But it can also be kept as `"core": false` and `"unlocked_by": null` :)

Let me know if any of that didn't make sense! :)

Reply:
It also needs a `uuid`. These were introduced quite recently, see [this PR](https://github.com/exercism/java/pull/704). You can use [this tool](https://www.uuidgenerator.net/) to generate one :)

Reply:
I just found [this document](https://github.com/exercism/docs/blob/master/language-tracks/configuration/exercises.md) which explains very well what needs to be in this file (including `uuid`, `unlocked_by` and `core`) :)